Refactoring towards webonyx/graphql-php.

fubhy opened the pull request.

fubhy committed:
Refactoring towards webonyx/graphql-php.

fubhy committed:
Reorganizing scalars.

fubhy committed:
Switch to webonyx server for handling batch and persisted queries.

fubhy committed:
Fixing access check.

fubhy committed:
Allow setting a context.

fubhy committed:
Refactor request controller.

pmelab committed:
Fixed result assertion trait.

pmelab committed:
Fixed result retrieval and disabled cache assertions for now.

fubhy committed:
Adding caching for plugin definitions.

sebas5384 commented:
@fubhy and @pmelab this means we are opening the door to be able to use files like `schema.graphql` to declare schema?

fubhy commented:
@sebas5384 yes, theoretically we can even look into generating our plugins based on that. Let's see.

pmelab committed:
Fixed serializaton error in test runs.

pmelab committed:
Don't throw on error.

fubhy committed:
Cleanup.

pmelab committed:
Test system fixes.

pmelab committed:
Mutations fix.

pmelab committed:
Mutations fix.

pmelab committed:
Inject mock plugins without requiring multiple managers per type.

pmelab committed:
Add missing method to MockTypeSystemPluginManager.

fubhy committed:
Adding comment.

fubhy committed:
Adding caching of schema builder.

fubhy committed:
Adding all types to the schema.

fubhy committed:
Statically cache plugin instances.

fubhy committed:
Override query processing logic.

pmelab committed:
Refactored plugin mock system.

fubhy committed:
Fixing cache lookup.

fubhy committed:
Better batch handling and starting to implement cache collector.

fubhy committed:
Better batch handling and starting to implement cache collector.

pmelab committed:
Fixed mutation cache.
Fixed result data accessors.

pmelab committed:
Adjusted to enum breaking change.

pmelab committed:
Fixed mutation resolve invocation.

pmelab committed:
Fixed enum test.

pmelab committed:
Implemented interface and union type resolving.

fubhy committed:
Simplifying type lookup.

fubhy committed:
Fixing tests.

fubhy committed:
Fixing tests.

fubhy committed:
Fixing tests.

fubhy committed:
Fixing tests.

fubhy committed:
Inherit interface fields.

fubhy committed:
Fixing mutations.

fubhy committed:
Re-adding file uploads.

fubhy committed:
Inherit interfaces.

fubhy committed:
Fixing mutation tests.

fubhy committed:
Fixing interfaces.

fubhy committed:
Fixing test.

pmelab committed:
Introspection based schema cache tests.

fubhy committed:
I am the queen of refactoring.

fubhy committed:
Use operation name in context.

fubhy committed:
Cast rendered output.

fubhy committed:
Post refactoring.

fubhy committed:
Refactoring processor.

pmelab committed:
Adapted field mocks to latest changes.

pmelab committed:
Improved error assertions. Output internal errors and search for regexes.

pmelab committed:
Fixed batched query and file upload tests.

pmelab committed:
Fixed batched permissions test.

pmelab committed:
Respect permission to bypass field security.

pmelab committed:
Respect type weights in association map.

pmelab committed:
Fixes for various core components.

pmelab committed:
Entity field fixes.

pmelab committed:
Entity field fixes.

pmelab committed:
Re-enabling cache assertions.

pmelab committed:
Improved node schema tests.

fubhy committed:
Working on cache metadata collection pattern.

fubhy committed:
Fixing tests.

fubhy committed:
Fixing tests.

fubhy committed:
Produce query hash.

fubhy committed:
Fixing hashing.

fubhy committed:
Adding cache context collection.

fubhy committed:
Fixing tests.

fubhy committed:
Add response cache tags and max age.

fubhy committed:
Adding missing contexts to error message.

fubhy committed:
Fixing cache dependencies.

fubhy committed:
Attempting to fix serialization.

fubhy committed:
Adding graphql_response cache tag.

fubhy committed:
Adding derived cache metadata.

fubhy committed:
Adding variables.

fubhy committed:
Mark queries ans uncachable.

fubhy committed:
Do not cache in development.

fubhy committed:
Move schema builder into plugin.

fubhy committed:
Fixing tests.

fubhy committed:
Fixing tests.

fubhy committed:
Fixing tests.

fubhy committed:
Remove test for caching in batched queries because batched queries run in parallel now.

fubhy committed:
Use deriver for collecting schema definitions.

pmelab committed:
Refactored mocking framework.

pmelab committed:
Avoid double-decoration of plugins during mocking.

pmelab committed:
Don't serialize result errors, since they may contain closures.

pmelab committed:
Fixed result cache test.

pmelab committed:
Fixed entity mutation tests.

fubhy committed:
Adding cache tag and max age collection in plugin managers.

pmelab committed:
Fixing expected cache contexts.

pmelab committed:
Fixed schema cache tests.
Merged schema cache metadata into response.
Fixed cache metadata test expectations.

fubhy committed:
Adding docblocks.

fubhy committed:
Removing redundant cache tags.
